### PR TITLE
[PAY-685] Update Stripe button to use approved brand assets

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Title. Will cherry-pick into `mjp-stripe-demo` when merged.

modal:
image

transaction details:
image

button default:
image

button hover:
image
image

darkmode:
image

matrix:
image

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-using-link

File: packages/web/src/store/application/ui/buy-audio/sagas.ts [hunk 1]
```diff
@@ -90,8 +90,8 @@ const SLIPPAGE = 3 // The slippage amount to allow for exchanges
 const BUY_AUDIO_LOCAL_STORAGE_KEY = 'buy-audio-transaction-details'
 
 const MEMO_MESSAGES = {
-  [OnRampProvider.COINBASE]: 'In-App $AUDIO Purchase: Coinbase',
-  [OnRampProvider.STRIPE]: 'In-App $AUDIO Purchase: Stripe',
+  [OnRampProvider.COINBASE]: 'In-App $AUDIO Purchase: Coinbase Pay',
+  [OnRampProvider.STRIPE]: 'In-App $AUDIO Purchase: Link by Stripe',
```
Comment (piazzatron) [inline packages/web/src/store/application/ui/buy-audio/sagas.ts:94]: could probs just keep this one stripe since it's only a memo 🤷‍♂️

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-using-link

**Review by piazzatron — APPROVED** [review]
LGTM! I was testing on the FE demo subdomain and see the Stripe button is disabled in my region. I thought it was NY at first but even when on a VPN elsewhere in US, it seems disabled?

**Pull request merged**